src/sentry/api/helpers/group_index.py

```diff
@@ -420,6 +420,22 @@ def self_subscribe_and_assign_issue(acting_user, group):
             return Actor(type=User, id=acting_user.id)
 
 
+def track_update_groups(function):
+    def wrapper(request, *args, **kwargs):
+        response = function(request, *args, **kwargs)
+
+        serializer = GroupValidator(data=request.data, partial=True)
+        results = dict(serializer.validated_data) if serializer.is_valid() else {}
+        tags = {key: True for key in results.keys()}
```

Comment:
Does mean we'll have a bunch of group ids tagged on this metric? That seems like it could potentially be a lot of data?

Reply:
I don't see any from testing (ie. doing updates and printing this dict) the group_ids are grabbed here instead: https://github.com/getsentry/sentry/pull/17542/files#diff-9577ad7c55d768a4da4a28e48d27c522R440

Reply:
Also worst case is 15 keys based on the `GroupValidator` but that should be rare, and if that does happen I'd be very curious why a user is setting them all 🤔